docs/pages/guides/local-app-development.mdx

```diff
@@ -0,0 +1,248 @@
+
+<Terminal cmd={['$ choco install -y microsoft-openjdk11']} />
+
+If you already have JDK installed on your system, we recommend JDK11.
+
+</Step>
+
+#### Install and configure Android Studio
+
+Android Studio allows installing Android SDK, Android SDK Platform, and Android Virtual Device. They are required to set up the Android development environment. Follow the [instructions on the Android Studio Emulator](/workflow/android-studio-emulator/) to install these tools.
+
+</Tab>
+
+</Tabs>
+
+</Collapsible>
+
+<Collapsible summary="iOS">
+
+> **warning** A macOS is required to build projects with native code for iOS. You can create a [development build](/develop/development-builds/introduction/) to install the build on an iOS device.
+
+To set up your development environment on macOS for iOS, you'll need to install Node.js, Watchman, Xcode, and CocoaPods.
+
+#### Prerequisites
+
+Use a package manager such as [Homebrew](https://brew.sh/) to install the following dependencies.
+
+#### Install dependencies
+
+<Step label="1">
+
+Install the LTS release of Node.js using a [version management tool](https://nodejs.org/en/download/package-manager) such as `nvm` or `volta`. If you have already installed Node.js on your system, make sure the version is 16 or above.
+
+</Step>
+
+<Step label="2">
+
+[Install Watchman](https://facebook.github.io/watchman/docs/install#macos) using a tool such as Homebrew:
+
+<Terminal cmd={['$ brew install watchman']} />
+
+</Step>
+
+<Step label="3">
+
+Install Xcode using the [App Store](https://apps.apple.com/us/app/xcode/id497799835?mt=12). This tool will also install the iOS Simulator. You will also need to [install Command Line Tools](/workflow/ios-simulator/#install-xcode-command-line-tools).
+
+</Step>
+
+<Step label="4">
+
+[Install CocoaPods](https://guides.cocoapods.org/using/getting-started.html). You can install using the version of Ruby that ships with the latest macOS version.
+
+</Step>
+
+</Collapsible>
+
+## Local app compilation
+
+To build your project locally you can use compile commands from Expo CLI which generates the **android** and **ios** directories:
+
+<Terminal
+  cmd={[
+    '# Build native Android project',
+    '$ npx expo run:android',
+    '# Build native iOS project',
+    '$ npx expo run:iOS',
+  ]}
+/>
+
+The above commands compile your project into an app locally. You can also add the `--device` flag to create a debug build of your project and run it on a connected device. These compilation commands initially run `npx expo prebuild` to generate native directories (**android** and **ios**) before building.
+
+To modify your project's configuration or native code after the first build, you've to rebuild your project. Running `npx expo prebuild` again layers the changes on the top of existing files. It may also produce different results after the build.
```

Comment:
```suggestion
To modify your project's configuration or native code after the first build, you will have to rebuild your project. Running `npx expo prebuild` again layers the changes on the top of existing files. It may also produce different results after the build.
```